cloud/docker_client.go

```diff
@@ -65,6 +65,10 @@ const (
 	imageImportTimeout  = 10 * time.Minute
 )
 
+func GetClient(s *evergreen.Settings) *dockerClientImpl {
```

Comment:
Other packages will call this like cloud.GetClient which is ambiguous which provider it's for. This should be something like GetDockerClient